app/modules/asset_groups/models.py

```diff
@@ -11,10 +11,15 @@
 import requests.exceptions
 import utool as ut
 
-from app.extensions import db, HoustonModel, parallel
 from app.extensions.gitlab import GitlabInitializationError
-from app.version import version
+from app.extensions import db, HoustonModel, parallel
+from app.modules.annotations.models import Annotation
```

Comment:
Why are these impports neeed if it's done in init.py?

Reply:
This import isn't for db.relationship, some code in asset_groups/models.py actually uses Annotation.

(Importing the models in app/module/__init__.py doesn't make Annotation available to asset_groups/models.py)

Reply:
I THINK JP was avoiding these top level includes due to dependency issues which is why they ended up being in the functions instead. 
Are you saying that what you've done in the init.py means that it is now safe to do this at the module level?

Reply:
Yeah I believe the problem was `db.relationship` model lookup errors:

```
sqlalchemy.exc.InvalidRequestError: When initializing mapper mapped class AssetGroupSighting->asset_group_sighting, expression 'Sighting' failed to locate a name ('Sighting'). If this is a class name, consider adding this relationship() to the <class 'app.modules.asset_groups.models.AssetGroupSighting'> class after both dependent classes have been defined.
```

Shouldn't be a problem anymore

Reply:
In which case, even better, thanks for clearing this up.

Reply:
By the way I did leave some of the imports at the function level, for example, "users" is the first ENABLED_MODULES so it can't import any other models...  but this is a pretty rare case and we can find it by starting the houston server